pkg/handlers/snapshots.go

```diff
@@ -176,6 +187,16 @@ func (h *Handler) UpdateGlobalSnapshotSettings(w http.ResponseWriter, r *http.Re
 		}
 	}
 
+	if updateGlobalSnapshotSettingsRequest.FileSystem != nil &&
```

Comment:
personal nit pick, this can be moved to the if statement above, like so:
```go
if updateGlobalSnapshotSettingsRequest.FileSystem != nil {
    if !globalSnapshotSettingsResponse.IsMinioDisabled {
         ....
    } else {
        ....
    }
}
```